paddle/fluid/operators/detection/distribute_fpn_proposals_op.cu

```diff
@@ -47,17 +47,8 @@ static inline int NumBlocks(const int N) {
                   kNumMaxinumNumBlocks);
 }
 
-static inline void TransLoD(const int* length_lod, const int lod_size,
-                            int* offset_lod) {
-  int offset = 0;
-  for (int i = 0; i < lod_size; ++i) {
-    offset_lod[i] = offset;
-    offset += length_lod[i];
-  }
-}
-
 template <typename T>
-static __device__ inline T RoIArea(const T* box, bool normalized) {
+__device__ T RoIArea(const T* box, bool normalized) {
```

Comment:
Do not copy code from other file, like RangeInitFunctor, CUDA_1D_KERNEL_LOOP,  RoIArea, ...

Reply:
Done, while CUDA_1D_KERNEL_LOOP is widely used in our cuda kernels